CHANGELOG.md

```diff
@@ -382,4 +382,4 @@ All notable changes to this project will be documented in this file. If a contri
 [v1.0.7]: https://github.com/styled-components/styled-components/compare/v1.0.6...v1.0.7
 [v1.0.6]: https://github.com/styled-components/styled-components/compare/v1.0.5...v1.0.6
 [v1.0.5]: https://github.com/styled-components/styled-components/compare/v1.0.4...v1.0.5
-[v1.0.4]: https://github.com/styled-components/styled-components/compare/v1.0.3...v1.0.4
+[v1.0.4]: https://github.com/styled-components/styled-components/compare/v1.0.3...v1.0.4
```

Comment:
The changelog is missing a *new* entry; Danger must've been thrown off by the accidental reformatting here :wink:

Reply:
@philpl I'm not sure why vscode went ahead and formatted this 😬 , do you want me to revert it?

Reply:
Yes please

Reply:
I pulled in the Changelog from my upstream/master branch to remove the formatting issues, let me know if this is ok.

Reply:
Fixed up the changelog and added a new entry.